src/test/java/de/retest/recheck/ui/descriptors/RootElementFilterTest.java

```diff
@@ -0,0 +1,224 @@
+package de.retest.recheck.ui.descriptors;
+
+import static de.retest.recheck.ui.Path.fromString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.AttributeFilter;
+import de.retest.recheck.ui.Path;
+import de.retest.recheck.ui.image.Screenshot;
+import de.retest.recheck.ui.image.Screenshot.ImageType;
+
+class RootElementFilterTest {
+
+	@Nested
+	class AttributeFilterTests {
+		private Element element;
```

Comment:
Why is this `element` private but `attributes` not? We don't expect test fields to be private, but either way pick one variant.